src/lib/viewers/doc/_docBase.scss

```diff
@@ -1,12 +1,30 @@
 @import '../../boxuiVariables';
 
 $thumbnail-border-radius: 4px;
+$thumbnail-sidebar-width: 201px;
 
 .bp {
     .bp-thumbnails-container {
         border-right: solid 1px $seesee;
+        bottom: 0;
         display: flex;
-        flex: 0 0 201px; // Accounts for the 1px border on the right so the remaining width is actually 180
+        flex: 0 0 auto;
+        left: 0;
+        position: absolute;
+        top: 0;
+        transform: translateX(-$thumbnail-sidebar-width);
+        transition: transform 300ms cubic-bezier(.4, 0, .2, 1);
+        width: $thumbnail-sidebar-width; // Accounts for the 1px border on the right so the remaining width is actually 180
```

Comment:
Should the comment live with the variable? Is the `180` part correct or should it be `200`?